packages/mui-system/src/styleFunctionSx/defaultSxConfig.js

```diff
@@ -0,0 +1,337 @@
+  pb: {
+    style: padding,
+  },
+  pl: {
+    style: padding,
+  },
+  px: {
+    style: padding,
+  },
+  py: {
+    style: padding,
+  },
+  padding: {
+    style: padding,
+  },
+  paddingTop: {
+    style: padding,
+  },
+  paddingRight: {
+    style: padding,
+  },
+  paddingBottom: {
+    style: padding,
+  },
+  paddingLeft: {
+    style: padding,
+  },
+  paddingX: {
+    style: padding,
+  },
+  paddingY: {
+    style: padding,
+  },
+  paddingInline: {
+    style: padding,
+  },
+  paddingInlineStart: {
+    style: padding,
+  },
+  paddingInlineEnd: {
+    style: padding,
+  },
+  paddingBlock: {
+    style: padding,
+  },
+  paddingBlockStart: {
+    style: padding,
+  },
+  paddingBlockEnd: {
+    style: padding,
+  },
+
+  m: {
+    style: margin,
+  },
+  mt: {
+    style: margin,
+  },
+  mr: {
+    style: margin,
+  },
+  mb: {
+    style: margin,
+  },
+  ml: {
+    style: margin,
+  },
+  mx: {
+    style: margin,
+  },
+  my: {
+    style: margin,
+  },
+  margin: {
+    style: margin,
+  },
+  marginTop: {
+    style: margin,
+  },
+  marginRight: {
+    style: margin,
+  },
+  marginBottom: {
+    style: margin,
+  },
+  marginLeft: {
+    style: margin,
+  },
+  marginX: {
+    style: margin,
+  },
+  marginY: {
+    style: margin,
+  },
+  marginInline: {
+    style: margin,
+  },
+  marginInlineStart: {
+    style: margin,
+  },
+  marginInlineEnd: {
+    style: margin,
+  },
+  marginBlock: {
+    style: margin,
+  },
+  marginBlockStart: {
+    style: margin,
+  },
+  marginBlockEnd: {
+    style: margin,
+  },
+
+  // display
+  displayPrint: {
+    cssProperty: false,
+    transform: (value) => ({
+      '@media print': {
+        display: value,
+      },
+    }),
+  },
+  display: {},
+  overflow: {},
+  textOverflow: {},
+  visibility: {},
+  whiteSpace: {},
+
+  // flexbox
+  flexBasis: {},
+  flexDirection: {},
+  flexWrap: {},
+  justifyContent: {},
+  alignItems: {},
+  alignContent: {},
+  order: {},
+  flex: {},
+  flexGrow: {},
+  flexShrink: {},
+  alignSelf: {},
+  justifyItems: {},
+  justifySelf: {},
+
+  // grid
+  gap: {
+    style: gap,
+  },
+  rowGap: {
+    style: rowGap,
+  },
+  columnGap: {
+    style: columnGap,
+  },
+  gridColumn: {},
+  gridRow: {},
+  gridAutoFlow: {},
+  gridAutoColumns: {},
+  gridAutoRows: {},
+  gridTemplateColumns: {},
+  gridTemplateRows: {},
+  gridTemplateAreas: {},
+  gridArea: {},
+
+  // positions
+  position: {},
+  zIndex: {
+    themeKey: 'zIndex',
+  },
+  top: {},
+  right: {},
+  bottom: {},
+  left: {},
+
+  // shadows
+  boxShadow: {
+    themeKey: 'shadows',
+  },
+
+  // sizing
+  width: {
+    transform: sizingTransform,
+  },
+  maxWidth: {
+    style: maxWidth,
+  },
+  minWidth: {
+    transform: sizingTransform,
+  },
+  height: {
+    transform: sizingTransform,
+  },
+  maxHeight: {
+    transform: sizingTransform,
+  },
+  minHeight: {
+    transform: sizingTransform,
+  },
+  size: {
+    style: (props) => {
+      const { size } = props;
```

Comment:
This looks new, we didn't have this shortcut before.

Why the addition? If it was for testing, I think to remove, and to add it as a demo.

Reply:
Oops this is my bad. I've added it because I saw these fined: https://github.com/mui/material-ui/blob/master/packages/mui-system/src/sizing.js#L49-L59 but they are not actually added in the [sizing](https://github.com/mui/material-ui/blob/master/packages/mui-system/src/sizing.js#L65) function. I am moving this to e demo, it's a nice show-case for adding new key 👍